go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/gokrazy/tools
 
 go 1.19
 
+replace github.com/gokrazy/internal => /Users/ddonati/src/private/internal
```

Comment:
This will need to change one https://github.com/gokrazy/internal/pull/16 merges.

Reply:
Not needed anymore, removed.